README.md

```diff
@@ -1 +1,20 @@
-see https://w3c.github.io/rdf-star/
+## Draft ##
+
+See https://w3c.github.io/rdf-star/
+
+## Preliminary implementations ##
+
+| SA | PG | Implementation | Notes - Documentation
+|----|----|----------------|----------------------
+|    |  + | AllegroGraph   | in the works - https://lists.w3.org/Archives/Public/public-rdf-star/2020Aug/0021.html
+|    |  x | AnzoGraph      | https://docs.cambridgesemantics.com/anzograph/v2.2/userdoc/lpgs.htm?Highlight=rdf
+|    |  x | BlazeGraph     | https://github.com/blazegraph/database/wiki/Reification_Done_Right
+|  x |    | GraphDB        | http://graphdb.ontotext.com/documentation/9.2/free/devhub/rdf-sparql-star.html
+|  x |    | Jena           | https://jena.apache.org/documentation/rdfstar/
+|  + |  + | N3             | deferred - https://github.com/w3c/N3/issues/27#issuecomment-644768502
```

Comment:
```suggestion
|  x |  + | EYE             | PG can be emulated by adding rules - https://github.com/josd/eye/
```
As mentioned in the discussion, N3 is not an implementation, it is another extension of RDF. EYE is an implemetation of N3 that now also implements RDF* -- AFAIK, RubyRDF also implements both.